server/game.py

```diff
@@ -367,8 +384,14 @@ def check_collisions(self):
                         if train.score > self.best_scores.get(train.nickname, 0):
                             self.best_scores[train.nickname] = train.score
                             self._dirty["best_scores"] = True
-                        # Update the last delivery time for this train
-                        self.last_delivery_times[train.nickname] = current_time
+                        # Update the last delivery tick for this train
+                        self.last_delivery_ticks[train.nickname] = self.current_tick
+
+    def get_delivery_cooldown_ticks(self):
+        standard_tickrate = 60.0  # Reference tickrate
```

Comment:
Use tick_rate from the config (or remove it from the config). Hardcoding a value here is bugs waiting to happen.